### PR TITLE
[500326] implementing TODOs

### PR DESCRIPTION
**Pull request by svetelna**

### CODE REVIEW

**Comment by svetelna** [comment]
Done

**Comment by svetelna** [comment]
c9252f1189875d583387219f87a3e06876e178cc - implemented vector operations
5183d7f7036a9b434ac5812ed9f0f38725391253 - added unit tests
48b9ceab89e29ec7ac8f171f6e2a45f52e9c5989 - "fix" build
0c4be44a334c1dac19760c197808e64d16b868b5 - updating README


0c478394275c6d1061daef221c477f7ecbcc9f94 - squashed commit

19755cf97e0b0341a0f4762afbafbbba33e336e1 - merging conflicts
c7b0c347ca9b818366e03281b500bc943a2bd586 - implementing TODOs
5956875cb058ab13acda8d7b7f81fa5297679fb1 - adding tests
830db55b5e6b2ba857ebd2baf41b0785b312f4b0 - fix tests

9e3ac0bc21c826874122f511b6ad550443fb78eb - squashed commit

**Comment by ph4r05** [comment]
Hi @svetelna !

Few things:
- Squash pls to one commit, there are 4 now
- Make sure the squashed commit is signed
- Travis is passing but with 0 tests executed. Pls take a look and fix it. 


Thanks!

**Comment by svetelna** [comment]
@ph4r05 Hello, is the PR correct now?

**Comment by ph4r05** [comment]
hi! no, unfortunately. You need to squash your commits to one. pls take a look at it.

tests are running, congrats! :)

**Comment by svetelna** [comment]
Could it be this way? :)

**Comment by ph4r05** [comment]
cool, thanks!

File: .travis.yml [hunk 1]
```diff
@@ -10,7 +10,6 @@ addons:
 
 python:
   - "3.6"
-  - "3.7"
```
Comment (ph4r05) [inline .travis.yml]: You could have added tests for 3.7, I mean, fix why 3.7 is not working not commenting it out :)

File: ossdev/__init__.py [hunk 2]
```diff
@@ -21,51 +24,167 @@ def set(self, arr):
...
         return self.d[item]
 
     def __hash__(self):
         return sum(self.d)
 
     def __setitem__(self, key, value):
-        # TODO: implement
-        return None
+        self.d[key] = value #done
+        return None
```
Comment (ph4r05) [inline ossdev/__init__.py]: Whitespace after None could have been removed

**Review by ph4r05 — DISMISSED** [review]
Nice code! 👍

File: .travis.yml (hunk 1 above)
Comment (svetelna) [reply .travis.yml]: I misunderstood, it is fixed properly now.

File: ossdev/__init__.py (hunk 2 above)
Comment (svetelna) [reply ossdev/__init__.py]: Also fixed.

**Review by ph4r05 — APPROVED** [review]

**Pull request closed without merging**